library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -0,0 +1,415 @@
+        position: relative;
+        > a {
+            text-decoration: none;
+            padding: 1px 7px 6px;
+            font-weight: bold;
+            font-size: 18px;
+            background: #fff;
+            border: 1px solid #555;
+            border-radius: 30px;
+            display: block;
+            line-height: 1;
+            position: absolute;
+            top: -10px;
+            right: -10px;
+        }
+    }
+}
+
+.criterion {
+    &.sakai-rubric-criterion,
+    &.sakai-rubric-criterion-preview,
+    &.sakai-rubric-criterion-grading,
+    &.sakai-rubric-criterion-student,
+    &.sakai-rubric-criterion-readonly {
+        width: 100%;
+        border-bottom: 1px solid #d3d3d3;
+        .criterion-row {
+            width: 100%;
+            height: 100%;
+            display: table;
+            border: 1px solid #d3d3d3;
+            border-width: 1px 0 0 0;
+            &.sortable-ghost, .sortable-ghost {
+                opacity: 0.25;
+            }
+            &.sortable-drag, .sortable-drag {
+                cursor: grabbing;
+                cursor: -webkit-grabbing;
+                cursor: -moz-grabbing;
+            }
+            .criterion-detail {
+                position: relative;
+                padding: 10px;
+                display: table-cell;
+                vertical-align: top;
+                background: #f2f2f4;
+                width: 20%;
+                border-right: 1px solid #d3d3d3;
+                p {
+                    margin: 0;
+                    padding: 0;
+                    font-size: 11px;
+                    color: #414141;
+                }
+            }
+            .reorder-icon {
+                cursor: grab;
+                cursor: -webkit-grab;
+                cursor: -moz-grab;
+                &:active {
+                    cursor: grabbing;
+                    cursor: -webkit-grabbing;
+                    cursor: -moz-grabbing;
+                }
+            }
+            .criterion-ratings {
+                .rating-item {
+                    sakai-rubric-criterion-rating-edit {
+                    visibility: hidden;
+                    }
+                    &:hover sakai-rubric-criterion-rating-edit, &.active sakai-rubric-criterion-rating-edit {
+                        visibility: visible;
+                    }
+                }
+            }
+
+            .add-criterion-item {
+                display: none;
+                opacity: 0.5;
+                position: absolute;
+                padding-top: 5px;
+                right: -10px;
+                top: 0;
+                bottom: 0;
+                width: 20px;
+                z-index: 5;
+                &:hover {
+                    opacity: 1;
+                }
+            }
+
+            &:hover, &.active {
+                .add-criterion-item {
+                    display: block;
+                }
+                sakai-rubric-criterion-edit {
```

Comment:
A few cases of potential class missing the starting period. (Unless it is a custom element)